src/vs/editor/common/config/editorOptions.ts

```diff
@@ -2732,16 +2732,12 @@ export enum ShowAiIconMode {
 export interface IEditorLightbulbOptions {
 	/**
 	 * Enable the lightbulb code action.
-	 * Defaults to true.
+	 * The three possible values are `off`, `on` and `onCode` and the default is `onCode`.
+	 * `off` disables the code action menu.
+	 * `on` shows the code action menu on code and on empty lines.
+	 * `onCode` shows the code action menu on code only.
 	 */
-	enabled?: boolean;
-
-	experimental?: {
-		/**
-		 * Highlight AI code actions with AI icon
-		 */
-		showAiIcon?: ShowAiIconMode;
-	};
+	enabled?: ShowLightbulbIconMode;
```

Comment:
Consider using `registerEditorSettingMigration` and migrate `true`->remove it, `false`->`off`.